Vagrantfile

```diff
@@ -26,28 +26,36 @@ end
 
 Vagrant.configure(2) do |config|
 
-    config.vm.box = "debian/jessie64"
+    # Prefer docker over virtualbox since it is listed first
+    # meaning that docker should run if no '--provider' is supplied
+    # when calling vagrant up
+    config.vm.provider "docker" do |d, configOverride|
+        d.build_dir = "."
+        d.pull = true
+        d.has_ssh = true
+
+        # Overrides for 'config' unique for docker
+        configOverride.ssh.username = "vagrant"
+        configOverride.ssh.password = "vagrant"
+    end
 
-    config.vm.provider "virtualbox" do |vb|
+    config.vm.provider "virtualbox" do |vb, configOverride|
```

Comment:
If we get Docker working as we want, I don't think we should support VBox anymore. If we won't use VBox ourselves it's very likely that it will break or become unstable etc, without us noticing it.